src/components/DisplayedAnimals.tsx

```diff
@@ -20,7 +21,7 @@ const DisplayedAnimals = () => {
           : (animals))
           .map((item) => (
             <div
-              key={Math.random()}
+              key={uuidv4()}
```

Comment:
Iesaku uuid neizsaukt uz mapošanu, bet lietot tikai pie datu izveides - respektīvi, kad Tu, piemēram, pievieno jaunu dzīvnieku, tad dzīvniekam paredzēt arī id property, kura izveidošanai tad arī izmantotu uuid. Pie mapošanas iesaku ņemt unikālu, bet nemainīgu key - kā, piemēram, animal name vai animal species nosaukums. Savādāk kādā brīdī var parādīties negaidītas problēmas, kā nupat bija arī man. :)

Reply:
Ā, sapratu. Sasodīts, es cerēju, ka tikšu pa vieglo cauri, vienkārši aizstājot Math.random ar uuid :)
Šajā gadījumā gan animal name, gan species var atkārtoties, bet, ja būtu jau sākotnēji izveidots uuid kā dzīvnieka id, tad tas būtu tas unikālais identifikators.

Reply:
To dzīvnieku id tagad nelabošu, bet prikola pēc nomainīšu atpakaļ uz Math.random, lai ir kaut kādas izmaiņas pēc komentāra.